udaykalra/portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -85,10 +89,29 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
   }
 
   /**
-   * Obtains parameter from Comments typing field.
+   * Gets count for comments to be shown.
    */
+  private int getCommentCount(HttpServletRequest request) {
+    // Get the input from the form.
+    String commentCountString = request.getParameter("comment-count");
+
+    // Convert the input to an int.
+    int commentChoice;
+    try {
+      commentChoice = Integer.parseInt(commentCountString);
+    } catch (NumberFormatException e) {
+      System.err.println("Could not convert to int: " + commentCountString);
```

Comment:
Good job validating this input/converting to an int. When I noticed you added a comment-count query parameter I was worried the (string) parameter might be put directly into a DB query, which would create SQL-injection vulnerabilities.